workflows/cloudify_system_workflows/tests/snapshots/tests_utils.py

```diff
@@ -1,79 +0,0 @@
-"""Snapshot utilities test cases."""
-
-import os
-import shutil
-import subprocess
-import unittest
-import tempfile
-
-from pytest import mark
-
-from cloudify_system_workflows.snapshots.utils import make_zip64_archive
-
-
-class MakeZip64Test(unittest.TestCase):
```

Comment:
Why deleted this?

Reply:
We've discussed this at the office. It was marked as skip by you a year ago and never ran since.